pio/st7789_lcd/st7789_lcd.pio

```diff
@@ -33,6 +33,11 @@ static inline void st7789_lcd_program_init(PIO pio, uint sm, uint offset, uint d
     sm_config_set_fifo_join(&c, PIO_FIFO_JOIN_TX);
     sm_config_set_clkdiv(&c, clk_div);
     sm_config_set_out_shift(&c, false, true, 8);
+	
+	// The pin muxes can be configured to invert the output (among other things
+    // and this is a cheesy way to get CPOL=1
```

Comment:
The comment states `CPOL=1` whereas that is actually `CPOL=cpol`. Perhaps could change to
```diff
-and this is a cheesy way to get CPOL=1
+and this is a cheesy way to set CPOL
```

Reply:
True.  But I took that comment directly from code already in the repo.  I think it means that it's a cheesy way to get inverted polarity (i.e. CPOL=1, vs. CPOL=0).

Reply:
Fair